### DISCUSSION TITLE
Make StructuralEdgeConnection object applicable to internal edge StructuralCurveEdge

### DISCUSSION THREAD

**Discussion by scia-nv** [StructuralAnalysisFormat/StructuralAnalysisFormat-Doc#22] [Ideas]
Extend the functionality of line support with the option to be applied also on internal edges.

Proposed:
Add column internal edge - reference to StructuralCurveEdge
Extend force action enum - On edge

Example:
![image](https://user-images.githubusercontent.com/81751485/151804485-33869238-ee29-411e-b3a9-73324d6b5e8f.png)

It will allow to export similar analysis setups:
![image](https://user-images.githubusercontent.com/81751485/151790011-275748b2-9726-4143-aea6-35e36fdb6d05.png)

SAF file:
[SAF_internal_edge_proposal.xlsx](https://github.com/StructuralAnalysisFormat/StructuralAnalysisFormat-Doc/files/7971128/SAF_internal_edge_proposal.xlsx)

**Comment by sonkaste17** [comment]
Hi @dschuermans, this extension of SAF would not be a breaking change? (adding new reference column and an enum value)

**Reply by dschuermans** [reply]
Hi @sonkaste17, from the top of my head iId say no.

We're adding a new reference column.
The SDK will remove StructuralEdgeConnection objects which have a reference to StructuralCurveEdge if the target SAF version doesn't support those kind of references.
